tuplex/codegen/include/IFailable.h

```diff
@@ -23,6 +41,8 @@ class IFailable {
     bool _succeeded;
     bool _silentMode; // don't issue warnings
     std::vector<std::tuple<std::string, std::string>> _messages; //! stores messages in silent mode
+    std::vector<CompileError> _typeError;
```

Comment:
maybe rename to _compileError? In the future this is not necessarily a typeError, right?

Reply:
Yes. Renamed to _compileErrors.